.github/workflows/ci.yml

```
@@ -0,0 +1,129 @@
+name: CakePHP CI
+
+on: [pull_request]
+
+jobs:
+  testsuite:
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        php-version: ['7.2', '7.3']
+        db-type: [sqlite, mysql, postgres]
+    name: PHP ${{ matrix.php-version }} & ${{ matrix.db-type }}
+
+    services:
+      redis:
+        image: redis
+        ports:
+          - 6379/tcp
+      postgres:
+        image: postgres
+        ports:
+          - 5432/tcp
+
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        fetch-depth: 1
+
+    - name: Install packages
+      run: |
+        # Without removing php-fpm extensions installation fails for some reason
+        sudo apt remove php${{ matrix.php-version }}-fpm
+        sudo apt install php${{ matrix.php-version }}-intl
+        sudo update-alternatives --set php /usr/bin/php${{ matrix.php-version }}
+        sudo update-alternatives --set phar /usr/bin/phar${{ matrix.php-version }}
+        sudo update-alternatives --set phar.phar /usr/bin/phar.phar${{ matrix.php-version }}
+        sudo phpdismod xdebug
+        sudo phpenmod apcu
+
+        # memcached installation fails without updating packages.
+        # Packages updating is not done earlier to avoid all wasting time as all
+        # php packages get updated when installing php-intl.
+        sudo apt update
+        sudo apt install memcached
+
+        sudo locale-gen da_DK
+        sudo locale-gen de_DE
+
+        if [[ ${{ matrix.php-version }} == '7.2' ]]; then sudo apt install php-pcov; fi
+
+    - name: Set ENV vars
+      run: |
+        if [[ ${{ matrix.db-type }} == 'sqlite' ]]; then export DB_DSN='sqlite:///:memory:'; fi
+        if [[ ${{ matrix.db-type }} == 'mysql' ]]; then export DB_DSN='mysql://root@127.0.0.1/cakephp_test?init[]=SET sql_mode = "STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION"'; fi
+        if [[ ${{ matrix.db-type }} == 'postgres' ]]; then export DB_DSN='postgres://postgres@127.0.0.1:${{ job.services.postgres.ports['5432'] }}/cakephp_test'; fi
+
+    - name: composer install
+      run: composer install
+
+    - name: Run PHPUnit
+      env:
+        REDIS_PORT: ${{ job.services.redis.ports['6379'] }}
+      run: |
+        if [[ ${{ matrix.php-version }} == '7.2' ]]; then
+          # Setting CODECOVERAGE is necessary to skip memory usage tests
+          CODECOVERAGE=1 vendor/bin/phpunit --verbose --coverage-clover=clover.xml
+        else
+          vendor/bin/phpunit
+        fi
+
+    - name: Submit code coverage data
+      run: |
```

```suggestion
      if: matrix.php-version == '7.2'
      run: |
```
I think this should remove skipped `Submit code coverage data` step from non 7.2 jobs.

A bit more readable than bash `if`.

---

It still does show the `Submit code coverage data` step for PHP 7.3 jobs.

---

At least it has white square icon to tell it skipped